src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int numberBall = 100;
```

Comment:
Is not a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)

Reply:
let's stick to the rule of naming constants.
You can see the link above